data/initrd/initrd.file_list

```diff
@@ -569,6 +570,10 @@ x etc/syslog.conf etc
 
 # additional scripts for linuxrc
 x scripts /
+# hmc exists only on s390
+if !exists(s390-tools-hmcdrvfs)
```

Comment:
I'm not sure if I like this compared to the previous check for an arch of s390 or s390x. What's the benefit of doing it this way?